merlin/systems/dag/ops/operator.py

```diff
@@ -41,22 +51,97 @@ def __repr__(self):
 
 
 class InferenceOperator(BaseOperator):
+    """
+    The basic building block of the Inference Node, the Inference Operator encapsulates
```

Comment:
My instinct is to either lowercase inference node and inference operator, or type like the code: `InferenceNode` and `InferenceOperator`

Ditto for ensemble.